pkg/jobs/jobspb/jobs.go

```diff
@@ -26,3 +26,21 @@ func (t *TraceData) ToText() []byte {
 	rec := tracingpb.Recording(t.CollectedSpans)
 	return []byte(rec.String())
 }
+
+// RestoreFrontierEntries is a slice of RestoreFrontierEntries.
+type RestoreFrontierEntries []RestoreProgress_FrontierEntry
```

Comment:
if y'all like this approach, I'll add a tiny unit test to validate this function works.